### PR TITLE
[Spring-Cloud]remove preview parameter and fix warning message

### PR DESCRIPTION
**Pull request by smile37773**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Spring-Cloud

File: src/spring-cloud/HISTORY.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 Release History
 ===============
+2.1.1
+-----
+* Remove preview parameter and fix warning message.
```
Comment (fengzhou-msft) [inline src/spring-cloud/HISTORY.md:5]: Can you add the parameter name as well? And it's only removed in `update`, right?

Comment (smile37773) [reply src/spring-cloud/HISTORY.md:5]: No, I just removed "--enable-java-agent" from "update" command, I didn't add new parameter.

Comment (fengzhou-msft) [reply src/spring-cloud/HISTORY.md:5]: I mean make the notes clear like `Remove preview parameter --enable-java-agent from az spring-cloud update`

Comment (smile37773) [reply src/spring-cloud/HISTORY.md]: ok,I see. fixed it.

**Review by fengzhou-msft — APPROVED** [review]

**Pull request merged**